falco.yaml

```diff
@@ -0,0 +1,121 @@
+package:
+  name: falco
+  version: 0.36.1
+  epoch: 0
+  description: Cloud Native Runtime Security
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - clang-16
+      - gcc
+      - cmake
+      - make
+      - git
+      - bash
+      - perl
+      - linux-headers
+      - autoconf
+      - automake
+      - m4
+      - libtool
+      - elfutils-dev
+      - libelf
+      - libelf-static
+      - patch
+      - binutils
+      - libbpf
+      - openssl
+      - openssl-dev
+      - yaml-dev
+      - c-ares
+      - c-ares-dev
+      - protobuf
+      - protobuf-dev
+      - protobuf-c-dev
+      - re2
+      - re2-dev
+      - zlib-dev
+      - libcurl-openssl4
+      - llvm16
+      - abseil-cpp
+      - abseil-cpp-dev
+      - abseillib
+      - jq-dev
+      - curl-dev
+      - grpc
+      - grpc-dev
+      - icu
+      - icu-dev
+      - yaml-cpp
+      - yaml-cpp-dev
+      - systemd-dev
+      - libsystemd
+      - libzstd1
+      - zstd
+      - zstd-dev
+      - libtbb-dev
+
+vars:
+  RULES_VERSION: "2.0.0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/falcosecurity/falco
+      tag: ${{package.version}}
+      expected-commit: 9eb611609a2876a5f5a5378e0613f0ff767f0d42
+      recurse-submodules: true
+
+  - runs: |
+      # Replace the find_dependency with find_package macro for newer cmake, otherwise it will fail.
+      sed -i 's/find_dependency(Protobuf CONFIG)/find_package(Protobuf CONFIG)/' /usr/lib64/cmake/grpc/gRPCConfig.cmake
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/etc/falco
```

Comment:
Same here as below, but is this something that we expect users to customize? Maybe turn this into subpackage also?